static/app/views/disabledMember/index.tsx

```diff
@@ -1,30 +1,10 @@
-import {RouteComponentProps} from 'react-router';
-
 import NotFound from 'app/components/errors/notFound';
 import HookOrDefault from 'app/components/hookOrDefault';
-import {OrganizationSummary} from 'app/types';
-import withOrganizations from 'app/utils/withOrganizations';
-
-type Props = {
-  organizations: OrganizationSummary[];
-} & RouteComponentProps<{orgId: string}, {}>;
 
 //getsentry will add the view
 const DisabledMemberComponent = HookOrDefault({
   hookName: 'component:disabled-member',
   defaultComponent: () => <NotFound />,
 });
 
-function DisabledMember(props: Props) {
-  const {
-    organizations,
-    params: {orgId},
-  } = props;
-  const org = organizations.find(o => o.slug === orgId);
-  if (!org) {
-    return <NotFound />;
-  }
-  return <DisabledMemberComponent organization={org} />;
-}
-
-export default withOrganizations(DisabledMember);
```

Comment:
Why is this no longer needed?

Reply:
@billyvg I realized that I needed the full `Organization` instead of `OrganizationSummary` so I opened up the organization detail route on getsentry for disabled members so we can load the organization from that endpoint directly. See: https://github.com/getsentry/getsentry/pull/5663/files#diff-3c70c2c6231b58fab461c75281e7667a0c0af7c138ac66f76c9e4c506a3598eaR28